### PR TITLE
Add tox.ini

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -6,7 +6,6 @@ ipython==0.10
 sqlparse==0.1.1
 selenium==1.0.1
 translate-toolkit==1.6.0
-pylint==0.20.0
 pygments==1.4
 pep8==0.5.0
 mock==0.6.0
```
Comment (groovecoder) [inline requirements/dev.txt]: We should add `tox==1.8.1`?

**Comment by groovecoder** [comment]
Changes looks good. :+1: getting rid of `pylint`!
I got this traceback when I tried to run `tox` on my vm. (After adding it to `requirements/dev.txt` and installing it):
```
Traceback (most recent call last):
  File "manage.py", line 36, in <module>
    import jingo.monkey
  File "/home/vagrant/src/vendor/src/jingo/jingo/__init__.py", line 160, in <module>
    env = get_env()
  File "/home/vagrant/src/vendor/src/jingo/jingo/__init__.py", line 48, in get_env
    loaders = [loader(p) for loader, places in x for p in places]
  File "/home/vagrant/src/.tox/py26/lib/python2.6/site-packages/jinja2/loaders.py", line 216, in __init__
    provider = get_provider(package_name)
  File "/home/vagrant/src/.tox/py26/lib/python2.6/site-packages/pkg_resources/__init__.py", line 396, in get_provider
    __import__(moduleOrReq)
  File "/home/vagrant/src/vendor/lib/python/dbgettext/__init__.py", line 42, in <module>
    autodiscover()
  File "/home/vagrant/src/vendor/lib/python/dbgettext/__init__.py", line 33, in autodiscover
    import_module("%s.%s" % (app, REGISTRATION_MODULE_NAME))
  File "/home/vagrant/src/vendor/src/django/django/utils/importlib.py", line 35, in import_module
    __import__(name)
  File "/home/vagrant/src/kuma/search/dbgettext_registration.py", line 4, in <module>
    from .models import Filter, FilterGroup
  File "/home/vagrant/src/kuma/search/models.py", line 18, in <module>
    from kuma.wiki.models import Document
  File "/home/vagrant/src/kuma/wiki/models.py", line 129, in <module>
    class DocumentTag(TagBase):
  File "/home/vagrant/src/kuma/wiki/models.py", line 131, in DocumentTag
    class Meta:
  File "/home/vagrant/src/kuma/wiki/models.py", line 132, in Meta
    verbose_name = _("Document Tag")
  File "/home/vagrant/src/vendor/src/tower/tower/__init__.py", line 20, in ugettext
    ret = django_ugettext(message)
  File "/home/vagrant/src/vendor/src/django/django/utils/translation/__init__.py", line 86, in ugettext
    return _trans.ugettext(message)
  File "/home/vagrant/src/vendor/src/django/django/utils/translation/trans_real.py", line 278, in ugettext
    return do_translate(message, 'ugettext')
  File "/home/vagrant/src/vendor/src/django/django/utils/translation/trans_real.py", line 268, in do_translate
    _default = translation(settings.LANGUAGE_CODE)
  File "/home/vagrant/src/vendor/src/django/django/utils/translation/trans_real.py", line 183, in translation
    default_translation = _fetch(settings.LANGUAGE_CODE)
  File "/home/vagrant/src/vendor/src/django/django/utils/translation/trans_real.py", line 160, in _fetch
    app = import_module(appname)
  File "/home/vagrant/src/vendor/src/django/django/utils/importlib.py", line 35, in import_module
    __import__(name)
ImportError: No module named devserver
ERROR: InvocationError: '/home/vagrant/src/.tox/py26/bin/coverage run manage.py test --noinput -v2 kuma'
```
The flake8 failed too, but it was expected.

File: requirements/dev.txt (hunk 1 above)
Comment (jezdez) [inline requirements/dev.txt]: Right, if that's where you want to use tox, sure.

**Comment by jezdez** [comment]
Bah, the `settings_local.py` is super annoying as it's implicitly adding devserver to the settings used by the tests. Gonna fix this for good soon..

**Comment by jezdez** [comment]
@groovecoder Fixed

**Pull request merged**